app/views/docs/command-line.phtml

```diff
@@ -1,67 +1,148 @@
+<p>The <a href="https://github.com/appwrite/sdk-for-cli" target="_blank" rel="noopener">Appwrite CLI</a> is a command-line application that allows you to interact with the <a href="/docs/getting-started-for-server">Appwrite server</a> and perform server-side tasks quickly using your terminal. This includes creating and managing projects, managing resources (documents, files, users), creating and deploying Cloud Functions, and other operations available through Appwrite's API.</p>
 
-<h2><a href="/docs/command-line#gettingStarted" id="gettingStarted">Getting started</a></h2>
+<h2><a href="/docs/command-line#installation" id="installation">Installation</a></h2>
 
-<p>The Appwrite CLI installation is extremely easy and fast. The CLI is packaged as a Docker container, which makes it secure by default, requires no dependencies but <a href="https://docs.docker.com/get-docker/">Docker</a>, and truly platform and language agnostic.</p>
+<p>The Appwrite CLI installation is extremely easy and fast. The CLI is packaged both as an <a href="https://www.npmjs.com/package/appwrite-cli">npm module</a> as well as a <a href="https://github.com/christyjacob4/appwrite-node-cli/releases/latest">standalone binary</a> for your operating system, making it completely dependency free, platform independent and language agnostic.</p>
 
-<p>The next step is to have your Appwrite server running. You can grab our simple <a href="/docs/installation">installation command</a> and execute it.</p>
+<p>The next step is to have your Appwrite server running. You can grab our simple <a href="/docs/installation">installation command</a> and execute it. Now, we're all set to play around with the CLI.</p>
 
-<p>Next, head over to <a href="http://localhost" rel="noopener" target="_blank">http://localhost</a> and create a project in the Appwrite Console, create an <a href="/docs/keys">API key</a>, and we're good to play around with the CLI.</p>
+<h3><a href="/docs/command-line#installWithNpm" id="installWithNpm">Install with npm</a></h3>
 
-<h3><a href="/docs/command-line#installation" id="installation">Installation</a></h3>
+<p>If you have npm set up, installing the CLI is as simple as running</p>
 
-<p>To install the CLI, we recommend running the installation script for your operating system.</p>
+<div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
+    <pre class="line-numbers">
+        <code class="prism language-bash" data-prism>
+            npm install -g appwrite-cli
+        </code>
+    </pre>
+</div>
+
+
+<h3><a href="/docs/command-line#installWithScript" id="installWithScript">Install with script</a></h3>
```

Comment:
Why have two options? Are developers going to choose to not use npm?

Reply:
So here was the thought process when we decided giving multiple installation options 
1. The previous version of the CLI was only distributed as a binary. So our install script was the only way for developers to install it. We didn't want to take away that option with the new release. 
2. While it might be safe to assume most developers have npm and node installed, we wanted to ensure our CLI can run without any dependencies.